.github/workflows/build.yml

```
@@ -100,8 +101,7 @@
       contents: write
       packages: write
     steps:
-      - name: Checkout code
-        uses: actions/checkout@v2
+      - uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)